pkgs/by-name/sd/sdl3-ttf/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  gitUpdater,
+  testers,
+  sdl3,
+  cmake,
+  freetype,
+  harfbuzz,
+  glib,
+  ninja,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sdl3-ttf";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "libsdl-org";
+    repo = "SDL_ttf";
+    tag = "release-${finalAttrs.version}";
+    sha256 = "0hp3r814866bn56m5q70m79b2pw92jj1v39vbabqfn6g1rdz5bks";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/libsdl-org/SDL_ttf/commit/ad2ffa8.patch";
+      sha256 = "1qyc0dfyyqang2dc2ma9gf3nhl2afafg33hga119jd6lbrbfrq58";
+    })
```

```suggestion
    (fetchpatch {
      url = "https://github.com/libsdl-org/SDL_ttf/commit/ad2ffa8.patch?full_index=1";
      hash = "sha256-emf7UnfB6Rl1+R74lsoIvm9ezDZtjHUS/t4k/RxbaYg=";
    })
```
sha256 -> hash. Also with fetching github diffs to patch, do `?full_index=1` or you run the risk of it not being reproducible afaik.